packages/components/htz-components/src/components/Image/DefaultImage.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from 'react';
+import { FelaComponent, } from 'react-fela';
+import type { ComponentPropResponsiveObject, } from '@haaretz/htz-css-tools';
+import { parseComponentProp, } from '@haaretz/htz-css-tools';
+import { aspectRatios, } from './Image';
+
+type Props = {
+  aspect?: string | ComponentPropResponsiveObject<string>[],
+  width?: ?number,
+  height?: ?number,
+};
+
+DefaultImage.defaultProps = {
+  aspect: null,
+  width: null,
+  height: null,
+};
+
+const style = ({ theme, aspect, width, height, }) => ({
+  backgroundColor: theme.color('neutral', '-5'),
+  height: 0,
+  extend: [ parseComponentProp('aspect', aspect, theme.mq, setAspect, width, height), ],
+});
+
+function setAspect(prop, aspect, width, height) {
+  const aspectRatio = width && height ? { width, height, } : aspectRatios[aspect];
+
+  const paddingPercentage = (aspectRatio.height / aspectRatio.width) * 100;
+  return {
+    paddingTop: `${paddingPercentage}%`,
+  };
+}
+
+export default function DefaultImage({ aspect, width, height, }: Props) {
+  if (!aspect && !(!width || !height)) {
```

Comment:
This should be:
```suggestion
  if (!aspect && !(width && height)) {
```

Which reads clearer to me, because it says, if `no aspect` and `no (width AND height)`, which is what we are checking. 

It is the same, but seems less convoluted to me, and the proliferation of bangs and the `or` operator obfuscate intent

Reply:
done